tests/sensu-client-windows.pp

```diff
@@ -0,0 +1,9 @@
+# Note: Check http://repositories.sensuapp.org/msi/ for the latest version.
+class { '::sensu':
+  version           => '0.26.5-1',
```

Comment:
i think we want `0.29.0-7` at http://repositories.sensuapp.org/msi/2012r2/sensu-0.29.0-7-x86.msi

You might be having troubles with such an old version.